apps/connections/index.jsx

```diff
@@ -91,4 +93,9 @@ var Connections = React.createClass({
   }
 })
 
-module.exports = Connections
+ipfsapp.define({
+  init: function (ipfs) {
+    React.render(<Connections ipfs={ipfs} />,
+                 document.getElementById('mount'))
+  }
+})
```

Comment:
I typically see (and do) code like

```
// If the module has a constructor 
exports = module.exports [= <name of func that might be a constructor or main call of this module>]

// and then, for other calls:
exports.anotherThing = function () { }
```

If the module is a facade, I do `exports.<something>` for all of them instead of doing `module.exports = {}` 

This way, if you can declare exported values along the file and you don't have to update the exported interface in two places.